src/plugins/intel_cpu/src/infer_request.cpp

```diff
@@ -197,10 +189,10 @@ void SyncInferRequest::change_default_ptr() {
         };
     }
 
-    for (auto& it : m_external_ptr) {
+    for (auto& it : m_input_external_ptr) {
         auto input = inputNodesMap.find(it.first);
         if (inputNodesMap.end() == input) {
-            OPENVINO_ASSERT(outputNodesMap.count(it.first), "Cannot find input/output blob: ", it.first);
+            OPENVINO_ASSERT(outputNodesMap.count(it.first), "Cannot find input/output blob with index: ", it.first);
             continue;
         }
```

Comment:
can we rewrite it to have `OPENVINO_ASSERT()` which check port is either in `inputNodesMap` or `outputNodesMap`?